status/views.py

```diff
@@ -0,0 +1,107 @@
+"""
+Status checks for LORE.
+
+Notes:
+
+* Useful messages are logged, but "no config found" is returned whether
+  settings are missing or invalid, to prevent information leakage.
+* Different services provide different information, but all should return
+  "up," "down", or "no config found" for the "status" key.
+"""
+
+import re
+from datetime import datetime
+import logging
+
+from django.conf import settings
+from django.http import JsonResponse
+from elasticsearch import Elasticsearch, ConnectionError as ESConnectionError
+from psycopg2 import connect, OperationalError
+from redis import StrictRedis, ConnectionError as RedisConnectionError
+
+log = logging.getLogger(__name__)
+
+
+def get_pg_info():
+    """Check PostgreSQL connection."""
+    try:
+        conf = settings.DATABASES['default']
+    except (AttributeError, KeyError):
```

Comment:
Are you saying that if everything else is functional, but Redis is down, you want a 500 error, and that you'd rather not know that the problem was Redis (or PostgreSQL or Elasticsearch)?

It seems that "something is broken" is not useful, and using one of the many free monitoring services would serve that purpose nicely. There's no point in this branch at all if that's the case. Knowing that the Elasticsearch server is down would give a much bigger head-start on fixing the problem.